[Ubuntu 20.04], [ansible 2.13.2]: Issue with apt fact processing leads to python apt packages not being processed

"Out of the box" install on Ubuntu 20.04, the pip install of uwsgi failed with...
```
      *** uWSGI compiling embedded plugins ***
      [x86_64-linux-gnu-gcc -pthread] plugins/python/python_plugin.o
      In file included from plugins/python/python_plugin.c:1:
      plugins/python/uwsgi_python.h:2:10: fatal error: Python.h: No such file or directory
          2 | #include <Python.h>
            |          ^~~~~~~~~~
      compilation terminated.
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for uwsgi
  Running setup.py clean for uwsgi
Failed to build uwsgi
```

Python package installation is being skipped:
```
Install python3 packages...
Tuesday 30 August 2022  16:23:28 +1000 (0:00:00.065)       0:00:20.855 ********
  openwisp2.local skipped: {
    "changed": false,
    "skip_reason": "Conditional result was False"
}
```

I _assume_ it's because [the fact check](https://github.com/openwisp/ansible-openwisp2/blob/master/tasks/apt.yml#L184) 
```
  when: openwisp2_should_install_python_37 == "False\n"
```
disagrees with 
```
Check openwisp2 should install python 3.7...
Tuesday 30 August 2022  16:23:28 +1000 (0:00:00.610)       0:00:20.436 ********
  openwisp2.local ok: {
    "ansible_facts": {
        "openwisp2_should_install_python_37": false
    },
    "changed": false
}
```

Change introduced here https://github.com/openwisp/ansible-openwisp2/commit/92da57657a88cdb08ab67222da75802ca287a486#diff-bd68d57a0208eff15ccf26bd0a9e052d8f76cc2b5305f2a544456a8a5f135e55R201

Am unsure if "False\n" was purely a typo, otherwise I'd just submit a PR

--- a/tasks/apt.yml
+++ b/tasks/apt.yml
@@ -181,7 +181,7 @@
   delay: 10
   register: result
   until: result is success
-  when: openwisp2_should_install_python_37 == "False\n"
+  when: openwisp2_should_install_python_37 == false
 
 - name: Install ntp
   when: openwisp2_install_ntp

